index.html

```diff
@@ -55,8 +55,8 @@ <h3>&#x21E1;</h3>
     <div class="header-text-container">
         <h1>WYSZUKIWARKA</h1>
         <h2 class="shadow">CZĘŚCI SAMOCHODOWYCH</h2>
-        <h3 class="shadow">Pomożemy Ci znaleźć <a class="marker">każdą część</a> do Twojego samochodu,<br/>a jako <a class="marker">BONUS</a> wskażemy Ci najbliższy
-        warsztat!</h3>
+        <h3 class="shadow">Pomożemy Ci znaleźć <a class="marker">każdą część</a> do Twojego samochodu,<br/>a jako <a class="marker">BONUS</a> zachowamy ją dla
+             Ciebie w Ulubionych!</h3>
         <h3 class="shadow">Startujemy już wkrótce!</h3>
```

Comment:
"a jako BONUS będziesz mógł zachować ją w ulubionych" ?
It's user who saves parts for himself, not app developers :D

Reply:
I mean that when user saves part, he does it by himself - it's not done by some other people. It sounded a bit strange to me ;)

Reply:
Pomożemy Ci znaleźć każdą część do Twojego samochodu, a jako BONUS zachowamy ją dla Ciebie w Ulubionych! - application saves the part for user thanks to the login. It also quite understandable.

Reply:
I don't think it is wrongly written. Kamila also approved this part so I guess it might stay as it is.

Reply:
Yep, looks good to me

Reply:
Ok, I have other opinion but I won't argue about that :)